app/gateway-manager/data-plane-reference.md

```diff
@@ -13,24 +12,213 @@ works_on:
 api_specs:
     - konnect/control-planes-config
 faqs:
-  - q: Will a new serverless gateway be provisioned in the same region as {{site.konnect_short_name}}?
-    a: Deployment on the same region is not guaranteed. 
+  - q: How can I verify that a Data Plane node is running?
+    a: |
+      You can verify a Data Plane node by accessing a configured route through its proxy URL. 
+      By default, {{site.base_gateway}} listens on port `8000`, so a request to `http://localhost:8000/{your-route}` (or your custom hostname) 
+      should return the expected response from your upstream service.
+
+  - q: How do I access services through a Data Plane node running on Kubernetes?
+    a: |
+      1. Run the following command to get the external IP and port:
+         ```bash
+         kubectl get service my-kong-kong-proxy -n kong
+         ```
+      2. Find the IP in the `EXTERNAL-IP` column and use it with port `80` or `443` along with your route.
+
+         For example, if the external IP is `35.233.198.16` and your route is `/mock`, access your service at:
+         ```
+         http://35.233.198.16:80/mock
+         ```
+  - q: Can I choose a specific {{site.base_gateway}} version when using Quickstart scripts in Gateway Manager?
+    a: |
+      Yes. Gateway Manager allows you to select the {{site.base_gateway}} version for your Quickstart scripts—except when using cloud provider scripts for AWS, Azure, and GCP.
```

Comment:
```suggestion
      Yes. Gateway Manager lets you select the {{site.base_gateway}} version for your quickstart scripts.
```
I don't think think we have cloud provider scripts anymore (I checked in konnect).